src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleFiltering/components/CollectionDateFilter/index.tsx

```diff
@@ -51,16 +56,19 @@ const CollectionDateFilter = ({
         onClick={handleClick}
       />
       <StyledDateFilterMenu
-        {...props}
         anchorEl={anchorEl}
+        data-test-id="collection-date"
+        fieldKeyEnd="collectionDateEnd"
+        fieldKeyStart="collectionDateStart"
+        menuOptions={[...MENU_OPTIONS_COLLECTION_DATE, MENU_OPTION_ALL_TIME]}
         onClose={handleClose}
         onStartDateChange={setStartDate}
         onEndDateChange={setEndDate}
         selectedDateMenuOption={selectedDateMenuOption}
         setSelectedDateMenuOption={setSelectedDateMenuOption}
-        updateDateFilter={updateDateFilter}
+        updateDateFilter={noop}
```

Comment:
I don't understand how this became a `noop`? I don't see a corresponding change to `DateFilterMenu`, so is it that we can now handle everything just with `onStartDateChange={setStartDate}` and equivalent endDate?

Reply:
`updateDateFilter` isn't used here.  This still works because it's calling `setStartDate` and `setEndDate` directly.  I didn't change the implementation here, but I did move some of the props and typescript expects `updateDateFilter` to be passed in.